chore(core): remove `km_core_keyboard_load` API (#12769)
keymanapp/keyman: core/docs/api/keyboards.md

-------------------------------------------------------------------------------

# km_core_keyboard_load_from_blob() {#km_core_keyboard_load_from_blob}

## Description

> Parse and load keyboard from the supplied blob and size and a pointer to the
> loaded keyboard into the out paramter.

Reviewer: I'd omit "and size" since `blob` and it's size belong together. Or if the intention is to mention all parameters we'll have to mention the keyboard name as well. Overall I find this sentence hard to understand. What about something like

```suggestion
Parse and load a keyboard from the supplied blob and return a pointer to the
loaded keyboard in the out parameter.
```

Reviewer 2: agreed, size isn't an optional parameter so is pretty clear
